powerfulseal/clouddrivers/aws_driver.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import boto3
+from . import AbstractDriver
+from ..node import Node, NodeState
+
+
+def create_connection_from_config():
+    """ Creates a new aws api connection """
+    conn = boto3.resource('ec2')
+    return conn
+
+def get_all_ips(instance):
+    """
+        Digs out all the IPs from the instance.private_ip_address field
+        of an AWS EC2 instances
+    """
+    output = []
+    output.append(instance.private_ip_address)
+    output.append(instance.public_ip_address)
+    return output
+
+#https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_InstanceState.html
+MAPPING_STATES_STATUS = {
+    "RUNNING": NodeState.UP,
+    "STOPPED": NodeState.DOWN,
+    "TERMINATED": NodeState.DOWN,
+}
+
+def server_status_to_state(status):
+    return MAPPING_STATES_STATUS.get(status['Name'].upper(), NodeState.UNKNOWN)
+
+def create_node_from_server(server, ip):
+    """ Translate AWS EC2 Instance representation into a Node object.
+    """
+    return Node(
+        id=server.id,
+        ip=ip,
+        az=server.placement['AvailabilityZone'],
+        name="",
+        state=server_status_to_state(server.state),
+)
+
+class AWSDriver(AbstractDriver):
```

Comment:
Like you suggested, it would be awesome to add test for the `powerfulseal/clouddrivers/aws_driver.py` module 👍